script/Ciel/Day 14/14_01A_CIEL14A_6.txt

```diff
@@ -276,62 +299,62 @@ That's it.
 [sha:41b84ad6e73a841dd384178791f52f21c19b8f3e]{
 -- Page 223, Offset 38300.
 -- もう何年も前に落ちたことのある、
-I sink into the deep, deep darkness,
+Just as I did many years ago―――
 }
 [sha:9908e81ca9b0a8b032a2bc69a3e60732eb1d4050]{
 -- Page 223, Offset 38301.
 -- 深い深い闇へと落ちていけた。
-that I once fell into many years ago.
+I sink once more into the deep, deep, darkness.
 }
 [sha:56d0ce0ab477ef658e8f0b73896ab973757e4f08]{
 -- Page 224, Offset 38302.
 -- “遠野くん、遠野くん……！”
-"Tohno-kun, Tohno-kun...!"
+"Tohno-kun, Tohno-kun!"
 }
-[sha:7a2096a0636cd53e87541ba2f2c3d3e51fd426f0]{
+[offset:38303]{
 -- Page 224, Offset 38303.
 -- ―――声が、聞こえる。
-―――I hear a voice.
+―――I hear... a voice...
 }
 [sha:251039b21b5b273657905dcc2c37971846383b51]{
 -- Page 225, Offset 38304.
 -- “遠野くん、そんな……どうして……！”
-"Tohno-kun, why... did you..."
+"Tohno-kun, you... why!"
 }
 [sha:c4c016c5db11b57b23daf601946ef1c78833f978]{
 -- Page 225, Offset 38305.
 -- ―――ほら。やっぱり思ったとおり。
-―――Come on. It's just as I thought.
+―――See. I knew this would happen.
 }
 [sha:c2edc2c9c30535527724f08311bbeb72a05e8ca0]{
 -- Page 226, Offset 38306.
 -- “……やだ。こんなの、わたしはイヤです……！”
-"...No. I don't want this...!"
+"No... I don't want this!"
```

Comment:
"I don't want this" said nobody ever when they were sad.

Kind too MUCH of a stock phrase perhaps, but:
>No... Not——not like this!